bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -738,6 +627,159 @@ public Image(Device device, ImageDataProvider imageDataProvider) {
 	init();
 }
 
+
+/**
+ * Upon zoom update, this method copies the image data and add disabled or grayed image depending on the flag
+ *
+ * @param device the device on which to create the image
+ * @param data the image data to create the image from (must not be null)
+ * @param flag the style, either <code>IMAGE_DISABLE</code> or <code>IMAGE_GRAY</code>
```

Comment:
This documentation is incorrect. The flag may have any value, just that the method has some specific behavior for these two values.

Reply:
Sorry this comment was for this review: 
I am kind of confused here, shouldn't I just mention in docs what values should be passed to this method for correct usage?

Reply:
That's correct, but you call the method also with other flags and that's also a correct usage, isn't it? The behavior is as follows: for these two flags, the image applies changes to the given image data, for other flags it does not (but still they are valid).